src/entry_points.py

```diff
@@ -0,0 +1,76 @@
+import pkg_resources
+import os
+import setuptools
+
+from typing import List
+
+from src import wheel
+
+
+def install_entry_points(extracted_whl_directory) -> List[str]:
+    try:
+        distribution = next(pkg_resources.find_distributions(extracted_whl_directory))
+    except StopIteration:
+        raise RuntimeError(f"Could not find in Python distribution in directory: {extracted_whl_directory}")
+
+    entry_points = distribution.get_entry_map("console_scripts").items()
+    bin_dir_path = os.path.join(extracted_whl_directory, "bin")
```

Comment:
Can we assume the bin directory does not already exist?

One solution is to add a bin directory in the parent of the wheel which will be the root of the repository. This will mimic the global bin of python.

I also wonder whether putting the py_binary in the root of the repository is possible to avoid exposing the names as mentioned below

Reply:
> One solution is to add a bin directory in the parent of the wheel which will be the root of the repository.

Yeah interesting idea. As in the root of `py_deps`?

Reply:
Yep, the folder structure is:

```
<repo_name>/
    bin/    <---- new dir here
    pypi__foo/
    pypi__bar/
    pypi__baz/
```

Then the targets would have a consistent name `@<repo>//bin:<binary>`, or, even just `@<repo>//:<binary>`.

Reply:
Yeh that's an interesting idea. I can definitely try that out. Probably will pick this PR up again on a future weekend.